File: app/src/main/java/com/example/capstona_a/data/Board.java
package com.example.capstona_a.data;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Board {
    private Long id;
    private String name;
    private String content;
}

File: app/src/main/java/com/example/capstona_a/data/Board.kt
package com.example.capstona_a.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


data class Board(
    @SerializedName("id")
    @Expose
    val id: Long,
    @SerializedName("name")
    @Expose
    val name: String,
    @SerializedName("content")
    @Expose
    val content: String,
)